### ISSUE TITLE
Proper handling of custom scalar serialization/deserialization errors

### ISSUE THREAD

**Issue by slavikovbasa** [evo-company/hiku#154]
Having a scheme:
```
class Long(Scalar):
    @classmethod
    def parse(cls, value: typing.Any) -> int:
        return int(value)

    @classmethod
    def serialize(cls, value: typing.Any) -> int:
        return int(value)

GRAPH = Graph([
    Root([
        Field("longStub", Long, lambda fs: [fs[0].options["id"]], options=[Option("id", Long)])
    ])
])
```

Such a query is possible and results in a ValueError being thrown:
```
{
  longStub(id:"asdf")
}
```

It is possible to catch errors in `parse/serialize` methods of `Scalar`, but there is no way to return any validation errors to a client.
Would be nice to either:
1. Having some custom error that can be thrown from serialization methods and process it in hiku accordingly;
2. Or implementing a separate validation method for `Scalar`

Possibly there is the same problem with enums, haven't checked though.

**Comment by kindermax** [comment]
So basically we need to error propagation mechanism in denormalization ?

**Comment by slavikovbasa** [comment]
not sure if denormalization is the best place to do it, maybe somewhere after parsing?

**Comment by kindermax** [comment]
Yep, in case when we are parsing arguments we should catch parse errors and report them to user.

My msg about denormalization makes sense when we are serializing scalars. Also being able to report errors from in denormalization phase needed for another case when value completion failed (because of null in not-nullable field) and we need to tell user that there is an error.
